build_projects/dotnet-host-build/project.json

```diff
@@ -6,7 +6,7 @@
   },
   "dependencies": {
     "NETStandard.Library": "1.6.0-rc3-24201-00",
-    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24201-00",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2-rc3-24201-00",
```

Comment:
Chatted offline, the bootstrap loop is non-ideal. Let's keep no Microsoft.NETCore.App